projects/storefrontlib/src/cms-components/myaccount/update-email/update-email.module.ts

```diff
@@ -30,9 +34,10 @@ import { FormErrorsModule } from '../../../shared/index';
         },
       },
     }),
+    UpdateEmailService
   ],
-  declarations: [UpdateEmailFormComponent, UpdateEmailComponent],
-  exports: [UpdateEmailComponent, UpdateEmailFormComponent],
+  declarations: [UpdateEmailComponent],
+  exports: [UpdateEmailComponent],
   entryComponents: [UpdateEmailComponent],
```

Comment:
We no longer need the entryComponents. 
I don't think there's a good case for export, but I might be wrong